web_responsive_pwa/static/src/json/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Odoo PWA",
+  "short_name": "Odoo PWA",
+  "icons": [{
+    "src": "/web_responsive_pwa/static/img/icons/icon-128x128.png",
+      "sizes": "128x128",
+      "type": "image/png"
```

Comment:
Is it possible to just use 1 SVG icon?

Reply:
It seems it's not: https://developer.mozilla.org/en-US/docs/Web/Manifest/icons